lib/fake_braintree/sinatra_app.rb

```diff
@@ -7,12 +7,16 @@
 require 'fake_braintree/address'
 require 'fake_braintree/payment_method'
 require 'fake_braintree/transaction'
+require 'fake_braintree/client_token'
+require 'fake_braintree/credit_card_serializer'
 
 module FakeBraintree
   class SinatraApp < Sinatra::Base
     set :show_exceptions, false
     set :dump_errors, true
     set :raise_errors, true
+    set :public_folder, File.dirname(__FILE__) + '/braintree_assets'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.